lib/ansible/plugins/action/copy.py

```diff
@@ -25,7 +25,7 @@
 import tempfile
```

Comment:
I think it is a bit harder to read (as the split-code was better documented), whereas `checksum=force` is quite cryptic. But if this allows it to get merged, I am all for it.